packages/playground/blueprints/src/lib/blueprint.ts

```diff
@@ -74,6 +74,7 @@ export type BlueprintDeclaration = {
 		wp: string | 'latest';
 	};
 	features?: {
+		icu?: boolean;
```

Comment:
Maybe `intl` instead? The name relates to internationalization and a PHP extension, whereas `icu` isn't that well known as a shortcut.